Adwaita/adw/settingsdialog.styles

```diff
@@ -287,7 +287,7 @@ settingsdialog.styles
 			render {}
 			render_bg
 			{
-				0="gradient_horizontal(x0-6, y0, x0+6, y1, window_bg_transparent, window_bg)"
+				0="gradient_horizontal(x0-6, y0, x0+6, y1, window_bg_transparent, window_bg_transparent)"
```

Comment:
This will remove the gradient entirely. It's there to mask the combobox text smoothly instead of just clipping the text when it overflows.

Also [all 4 instances](https://github.com/tkashkin/Adwaita-for-Steam/search?q=gradient_horizontal) of `gradient_horizontal` should be verified and fixed. If one of them is using wrong colors there's a chance the other ones are using wrong colors as well.

Reply:
Doh! It was a theme issue, I was missing the override for the second half of the gradient.